rust/src/storage/file/rename.rs

```diff
@@ -119,8 +119,11 @@ mod tests {
                     }
                 }
             }
-            Err(e) => panic!(format!("expect std::io::Error, got: {:#}", e)),
-            Ok(()) => panic!("expect rename to fail with Err, but got Ok"),
+            Err(e) => {
+                let error_message = format!("expect std::io::Error, got: {:#}", e);
+                panic!("{}", error_message)
```

Comment:
same here, we can just use `panic!` like `format!`.

Reply:
👍